examples/lock-app/nrfconnect/README.md

```diff
@@ -101,135 +100,116 @@ the nRF52840's NFC antenna. Moreover in the reaction on sensing field from the
 smartphone CHIP device will start BLE advertisement, what is an alternative way
 to trigger this from manually pressing Button 4.
 
-<a name="building"></a>
+## Setting up environment
```

Comment:
I removed this: "The example makes use of the CMake build system to generate the ninja build script. The build system takes care of invoking the CHIP library build with all necessary flags exported from the Zephyr environment." from the introduction in my PR, as I thought it's not the best place for it, so I think that we should add this information somewhere in the setting up or building section.